java-checks/src/main/resources/org/sonar/l10n/java/rules/squid/S3753_java.html

```diff
@@ -0,0 +1,39 @@
+<p>A Spring <code>@Controller</code> that uses <code>@SessionAttributes</code> is designed to handle a stateful / multi-post form. Such
+<code>@Controller</code>s use the specified <code>@SessionAttributes</code> to store data on the server between requests. That data should be cleaned
+up when the session is over, but unless <code>setComplete()</code> is called on the <code>SessionStatus</code> object from a
+<code>@RequestMapping(method = POST)</code> method, neither Spring nor the JVM will know it's time to do that. Note that the
```

Comment:
I have removed "(method = POST)" from the RSPEC, because SessionStatus can also be used in a GET request. I added a comment to the RSPEC to explain my view about this rule implementation.